listener.go

```diff
@@ -70,7 +75,13 @@ func (l *listener) Accept() (tpt.CapableConn, error) {
 			sess.CloseWithError(0, err.Error())
 			continue
 		}
-		if l.transport.gater != nil && !(l.transport.gater.InterceptAccept(conn) && l.transport.gater.InterceptSecured(n.DirInbound, conn.remotePeerID, conn)) {
+		connScope, err := l.rcmgr.OpenConnection(network.DirInbound, false)
```

Comment:
where is this passed?

Reply:
Fixed by using the new `transport.CapableConn` interface.